ams_flutter/lib/features/onboarding/presentation/pages/splash.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:async';
+import 'package:ams_flutter/features/onboarding/presentation/pages/whoareyou.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class SplashScreen extends StatefulWidget {
+  @override
+  _SplashScreenState createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(
+        const Duration(seconds: 3),
+        () => Navigator.pushReplacement(context,
+            MaterialPageRoute(builder: (context) =>  WhoAreYou())));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        width: double.infinity,
+        height: double.infinity,
+
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Image.asset(
+              "Assets/images/splash_screen.png",
```

Comment:
It would be preferable if you were to create a variable under `images.dart` in `core/constants/` directory.

Reply:
Done